pkgs/by-name/rq/rquickshare/package.nix

```diff
@@ -0,0 +1,22 @@
+{ appimageTools, fetchurl, lib }:
+
+let
+  version = "0.7.1";
+in
+appimageTools.wrapType2 {
+  name = "rquickshare";
+  src = fetchurl {
+    url = "https://github.com/Martichou/rquickshare/releases/download/v${version}/r-quick-share_${version}_amd64.AppImage";
+    hash = "sha256-716d7T4nbs/dDS4KVGTADCpLO31U8iq6hDVD+c7Ks1I=";
+  };
+  # extraPkgs = pkgs: with pkgs; [ ]; # no extra packages required
+  meta = with lib; {
+    description = "A Quickshare client in Rust";
+    homepage = "https://github.com/Martichou/rquickshare";
+    changelog = "https://github.com/Martichou/rquickshare/blob/master/CHANGELOG.md#071-2024-05-01"; # edit anker with each release
+    license = licenses.gpl3;
```

Comment:
Don't write `licenses.gpl3` as this is deprecated. I've asked them here https://github.com/Martichou/rquickshare/issues/29#issuecomment-2118786273

Reply:
which one should be used instead?
isnt gpl3 the newest of the gpl family?